stdlib/contrib/sranka/webexteams/webexteams.flux

```diff
@@ -35,10 +37,7 @@ message = (
 // `token` - string - [Webex API access token](https://developer.webex.com/docs/api/getting-started).
 // The returned factory function accepts a `mapFn` parameter.
 // The `mapFn` must return an object with `roomId`, `text` and `markdown` properties  as defined in the `message` function arguments.
-endpoint = (
-        url = "https://webexapis.com",
-        token,
-) => (mapFn) => (tables=<-) => tables
+endpoint = (url="https://webexapis.com", token) => (mapFn) => (tables=<-) => tables
```

Comment:
Were the changes in this file just formatter changes? They don't seem relevant to the rest of the PR.

Reply:
Yes this was formatting